### DISCUSSION TITLE
Need Feedback and clarification on which Website platform should I use

### DISCUSSION THREAD

**Discussion by preetk9** [community/community#126066] [Programming Help]
### Select Topic Area

Question

### Body

I'm a non technical person who seeking a feedback from this technical world. I want to make a ecom website but not sure which platform I should use . As per my little knowledge and reached i came down to following three on which I need your feedback what should I do , 
1. MERN Stack 
2. Shopify for backend with frontend on different platforms like React/Angular 
3. The last one WordPress backend with frontend on different platforms like React/Angular.
Out of the three whichever would the best as per my requirement which is 
We need something which is highly efficient,scalable, secure and its backend frameworks are made in a way that a non technical person can use as well and also made in a way that if any small changes in design are to be made then a non technical person can also do it from its backend.
1. Hosting we already have from hostinger which is cloud plan
2. Domain 
3. SSL is also there with hostinger 
4. Google workplace for emails .
Considering all the above points which one I should choose as per my requirement and which cost cost effective

**Comment by AsmitaMishra24** [comment]
Hey @preetk9! 
Let’s evaluate the three options you’ve mentioned based on efficiency, scalability, security, ease of use, and cost-effectiveness:
- Shopify: Easy to use, secure, and scalable. Great for non-technical users. Monthly subscription cost.
- WordPress + WooCommerce: Customizable, cost-effective, but requires some technical knowledge.
- MERN Stack: Powerful but needs technical expertise. Custom development can be costly.

Recommendation:
If you're looking for a platform that's easy to use, secure, scalable, and allows for some customization without needing deep technical knowledge, go for Shopify with a React/Angular frontend. It strikes a good balance between user-friendliness, functionality, and cost-effectiveness for your e-commerce website.

**Reply by AsmitaMishra24** [reply]
I believe this will be helpful!

**Reply by preetk9** [reply]
Thanks

**Reply by AsmitaMishra24** [reply]
Hey @preetk9,
If this was helpful, please mark the answer as answered.
